examples/machine_translation/train.py

```diff
@@ -0,0 +1,156 @@
+import math
+import time
+
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from torch.utils.data.dataloader import DataLoader
+from tqdm import tqdm
+
+from dataset import get_dataset
+from embedding import WordCharCNNEmbedding
+from model import Attention, Decoder, Encoder, Seq2Seq
+from torchtext.data.metrics import bleu_score
+from torchtext.vocab import Vocab
+from utils import collate_fn, count_parameters, epoch_time, seed_everything
+
+# Ensure reproducibility
+seed_everything(42)
```

Comment:
We should allow users to set up the see, like [this](https://github.com/pytorch/text/blob/441edff8743cc03d2d8676029f5bea3c0ed76486/examples/BERT/mlm_task.py#L240)

Reply:
merged with other hyperparams with argparse